pandas/core/nanops.py

```diff
@@ -36,6 +37,7 @@ def set_use_bottleneck(v=True):
         _USE_BOTTLENECK = v
 
 
+
```

Comment:
you can remove this blank line? (in general, please try to avoid such changes on lines that you are not changing for this PR)